.github/CODEOWNERS

```diff
@@ -56,6 +56,7 @@ gradlew.bat @BenHenning
 # All resource files.
 /app/src/main/res/**/*.xml @rt4914
 /utility/src/main/res/**/*.xml @rt4914
+/app/src/main/res/values/colors_migrating.xml @ayush0402
```

Comment:
@ayush0402 Please also add me to this file.

Reply:
Is this correct now?